File: catkin_ws/python_ws/src/python_scripts/src/pub.py
#!/usr/bin/env python
import rospy
from std_msgs.msg import String

def main():
    rospy.init_node('publicador')
    pub = rospy.Publisher('publicacion', String, queue_size=1000)
    rate = rospy.Rate(1)

    count = 0
    rospy.loginfo('Practica P-S')

    while not rospy.is_shutdown():
        msg = String()
        p_string = 'Cuenta: ' + str(count)
        msg.data = p_string
        rospy.loginfo(msg.data + ' seg')
        pub.publish(msg)
        rate.sleep()
        count += 1

if __name__ == '__main__':
    try:
        main()
    except:
        pass

File: matlab/pub.m
publicacion_pub = rospublisher("publicacion", "std_msgs/String");
loop_rate = rosrate(1);

count = 0;
disp("PRACTICA P-S")

while true
    msg = rosmessage("std_msgs/String");
    msg.Data = "Cuenta: " + count;
    disp(msg.Data + " seg");
    send(publicacion_pub, msg);
    waitfor(loop_rate);
    count = count + 1;
end

